app/controllers/session_controller.rb

```diff
@@ -0,0 +1,21 @@
+class SessionController < ApplicationController
+  include SessionHelper
+  def new
+  end
+
+  def create
+    user = User.find_by email: params[:session][:email].downcase
+    if user && user.authenticate params[:session][:password]
+      log_in user
+      redirect_to user
+    else
+      flash[:danger] = "Invalid email/password combination"
+      render :new
+    end
+  end
+
+  def destroy
+    log_out
```

Comment:
trong ham login  da xu ly r ma c, khi login thanh cong thi sesson ms co gia tri, nen viec logout ms co the thuc hien duoc chu, ham destroy nhu vay da the hien la viec logout chi xay ra khi session co gia tri